scala-thoughts.md

```diff
@@ -0,0 +1,9 @@
+I'm not really sure if I have a judgement yet in the sense of whether or not I "like" Scala. Overall, I have a few mixed opinions, and some initial impressions, but I have intentionally tried not to form an opinion too soon, because I think I might be slightly biased by some external factors. Primarily, Scala was a huge pain to install and use (on Windows), so I'm not sure how I'd actually set up a scala project or script on my own. Second, and relatedly, I really dislike eclipse - its slow, terrible to look at, and has really unhelpful error messages sometimes. Since these things aren't actually issues with the language itself, I don't want to form an impression too soon.
+
```

Comment:
You are brave for installing Scala on windows haha. I tried for about 30 minutes and did it on my Mac.